ApollosProject/apollos-apps
# Adds VerticalCardListFeature

## DESCRIPTION

### What does this PR do, or why is it needed?

Companion PR: https://github.com/ApollosProject/apollos-templates/pull/17

This PR does the following: 

- Adds a VerticalCardListFeature
- Sets up the existing algorithms to be more generic.
- On the template side (see the companion PR) adds the corresponding front end components. 

In general, this gets up to the top of the hill for most of the home feed feature work. 

![2020-03-24 15 45 42](https://user-images.githubusercontent.com/1637694/77470142-9781e700-6de6-11ea-88ce-e8b25542547c.gif)

### What design trade-offs/decisions were made?

Trade offs captured in basecamp.

### How do I test this PR?

1. Pull this PR and it's companion PR
2. Start and link the app. 
3. Visit the home feed, you should see a content card nested between two other features. 

### What automated tests did you write?


## TODO:


## REVIEW:

**Manual QA**


**Code Review: Questions to consider**


> The purpose of PR Review is to _improve the quality of the software._
